packages/vue-cli-plugin-vuetify-preset-basil/preset/variables.scss

```diff
@@ -0,0 +1,37 @@
+$lekton-font-family: 'Lekton', sans-serif;
+$border-radius-root: 0;
+$body-font-family: 'Montserrat', sans-serif;
+$btn-font-weight: 700;
+$btn-icon-font-size: 0.875rem;
+$btn-outline-border-width: 2px;
+$headings: (
+  'h1': ('weight': 600),
+  'h2': ('weight': 600),
+  'h3': ('weight': 600),
+  'h4': ('weight': 600),
+  'h5': ('weight': 500),
+  'h6': (
+    'font-family': $lekton-font-family,
+    'weight': 700
+  ),
+  'subtitle-1': (
+    'font-family': $lekton-font-family,
+    'size': 1.0625rem,
+    'weight': 700
+  ),
+  'subtitle-2': (
+    'font-family': $lekton-font-family,
+    'size': 0.9375rem,
+    'weight': 700
+  ),
+  'body-1': ('weight': 600),
+  'body-2': ('weight': 600),
+  'caption': ('weight': 500)
+);
+$material-light: (
+  'background': #FFFBE6,
+  'cards': #FFFBE6
+);
+
+@import '~vuetify/src/components/VBtn/_variables.scss';
```

Comment:
Pretty sure you don't actually need this

Reply:
I'll take another look, but it was this way when I wrote the current documentation on it: https://vuetifyjs.com/en/customization/sass-variables#vue-cli-3-install